requirements/test-requirements.txt

```diff
@@ -17,8 +17,6 @@ matplotlib
 shap>=0.40
 plotly
 kaleido
-# The latest flask version requires werkzeug>=2.2.0
-werkzeug>=2.2.0
```

Comment:
moto unpinned werkzeug: https://github.com/spulec/moto/pull/5569

Reply:
https://github.com/pallets/flask/blob/3dc6db9d0cfddcfb971c382b014bb56ac3761d3c/setup.py#L7

```
        "Werkzeug >= 2.2.2",
```

Flask restricts werkzeug to ` >= 2.2.2`.